parsons/ngpvan/activist_codes.py

```diff
@@ -44,3 +41,60 @@ def get_activist_code(self, activist_code_id):
         r = self.connection.get_request(f'activistCodes/{activist_code_id}')
         logger.info(f'Found activist code {activist_code_id}.')
         return r
+
+    def toggle_activist_code(self, id, activist_code_id, action, id_type='vanid'):
```

Comment:
@jburchard just to be clear - will toggle still work? I know Yotam has it in code.

Reply:
Yes, it does, though is no longer surfaced in the docs. It is a deprecated method.